src/MockPlayerBridge.ts

```diff
@@ -49,12 +163,100 @@ export class MockPlayerBridge extends AbstractPlayerBridge {
     return !!document.fullscreenElement;
   }
 
+  protected subscribeMode(listener: (mode: PlaybackMode) => void) {
+    listener('LIVE');
+    return 'LIVE' as PlaybackMode;
+  }
+
+  protected subscribeMuted(listener: (value: boolean) => void) {
+    document.addEventListener('livery-muted-change', () => {
+      listener(this.muted);
+    });
+    return this.muted;
+  }
+
+  protected subscribePaused(listener: (value: boolean) => void) {
+    const pausedStates: PlaybackState[] = ['ENDED', 'PAUSED'];
```

Comment:
This should be implemented in AbstractPlayerBridge private subscribePaused() as mentioned above. Same goes for subscribePlaying and subscribeStalled below.